<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Juego Gatiano</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/gato-title.png</normaloff>:/gato-title.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_tablero">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>261</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/tablero_svg.svg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_sel_opcion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>9</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_elige">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Elige sabiamente&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonX">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonO">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>O</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOK">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_gato_logo">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/gato-negro.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>374</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu_Gatiano">
    <property name="title">
     <string>Menu Gatiano</string>
    </property>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuAcerca">
    <property name="title">
     <string>Acerca</string>
    </property>
    <addaction name="actionJuego_Gatiano"/>
    <addaction name="actionQt_Framwork"/>
    <addaction name="separator"/>
    <addaction name="actionFreepik_2"/>
   </widget>
   <addaction name="menuMenu_Gatiano"/>
   <addaction name="menuAcerca"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSair">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionJuego_Gatiano">
   <property name="text">
    <string>Juego Gatiano</string>
   </property>
  </action>
  <action name="actionQt_Framwork">
   <property name="text">
    <string>Qt Framwork</string>
   </property>
  </action>
  <action name="actionFreepik">
   <property name="text">
    <string>Freepik</string>
   </property>
  </action>
  <action name="actionFreepik_2">
   <property name="text">
    <string>Freepik</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
